packages/tools/workload-testing-utils/scripts/workload-testing-utils.js

```diff
@@ -110,69 +104,21 @@ export function getParent(lookupStartNode, path) {
     return parent;
 }
 
-/**
- * Page
- * 
- * Represents the environment of the workload, with a reference to the current document.
- * This class provides some utility functions to simplify targeting of DOM elements.
- */
-export class Page {
-    constructor(document) {
-        this.document = document;
-        this.body = document.body;
-    }
-
-    forceLayout() {
-        const rect = this.body.getBoundingClientRect();
-        const e = this.document.elementFromPoint(
-            (rect.width / 2) | 0,
-            (rect.height / 2) | 0
-        );
-        return e;
-    }
+export function getElement(selector, path = [], lookupStartNode = document) {
+    const element = getParent(lookupStartNode, path).querySelector(selector);
+    return element;
+}
```

Comment:
replacement for Page.querySelector()

Reply:
contemplating whether this should be just querySelector or queryElement ... (anyway bokeshedding).

Reply:
we should create a twitter poll 🤣